akka-remote/src/test/scala/akka/remote/artery/tcp/ssl/PeerSubjectVerifierSpec.scala

```diff
@@ -52,8 +52,7 @@ class PeerSubjectVerifierSpec extends AnyWordSpec with Matchers {
       override def removeValue(name: String): Unit = throw new UnsupportedOperationException()
       override def getValueNames: Array[String] = throw new UnsupportedOperationException()
       override def getLocalCertificates: Array[Certificate] = throw new UnsupportedOperationException()
-      override def getPeerCertificateChain /*: Array[javax.security.cert.X509Certificate]*/ =
-        throw new UnsupportedOperationException()
+      override def getPeerCertificateChain = throw new UnsupportedOperationException()
```

Comment:
This used to be `override def getPeerCertificateChain: Array[javax.security.cert.X509Certificate]` - are we sure we want to drop that explicit return type?

The downside is that javax.security.cert.X509Certificate has been deprecated since Java 9. Perhaps we can `@silence` that though?

Reply:
This is test code, I don't see any way how it could go wrong.